### ISSUE TITLE
Error in PhpStorm 2017.3 EAP 173.3188.26

### ISSUE THREAD

**Issue by Hunternnm** [kalessil/phpinspectionsea#581]
Hi. After the update, I started to get an error:
`com.jetbrains.php.lang.psi.elements.Method.getReturnType()Lcom/intellij/psi/PsiElement;`
PhpStorm 2017.3 EAP 173.3188.26

**Comment by kalessil** [comment]
Thank you for reporting @Hunternnm, can you share complete stack trace please?

**Comment by Hunternnm** [comment]
@kalessil oh, I apologize, hurried and forgot to insert

```
com.jetbrains.php.lang.psi.elements.Method.getReturnType()Lcom/intellij/psi/PsiElement;
java.lang.NoSuchMethodError: com.jetbrains.php.lang.psi.elements.Method.getReturnType()Lcom/intellij/psi/PsiElement;
	at com.kalessil.phpStorm.phpInspectionsEA.inspectors.languageConstructions.ReturnTypeCanBeDeclaredInspector$1.visitPhpMethod(ReturnTypeCanBeDeclaredInspector.java:72)
	at com.jetbrains.php.lang.psi.elements.impl.MethodImpl.accept(MethodImpl.java:85)
	at com.intellij.codeInspection.InspectionEngine.acceptElements(InspectionEngine.java:82)
	at com.intellij.codeInspection.InspectionEngine.createVisitorAndAcceptElements(InspectionEngine.java:70)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.a(LocalInspectionsPass.java:285)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.a(LocalInspectionsPass.java:253)
	at com.intellij.concurrency.ApplierCompleter.d(ApplierCompleter.java:133)
	at com.intellij.concurrency.ApplierCompleter.c(ApplierCompleter.java:223)
	at com.intellij.concurrency.ApplierCompleter.d(ApplierCompleter.java:151)
	at com.intellij.concurrency.ApplierCompleter.c(ApplierCompleter.java:223)
	at com.intellij.concurrency.ApplierCompleter.d(ApplierCompleter.java:151)
	at com.intellij.concurrency.ApplierCompleter.c(ApplierCompleter.java:223)
	at com.intellij.concurrency.ApplierCompleter.d(ApplierCompleter.java:151)
	at com.intellij.concurrency.ApplierCompleter.c(ApplierCompleter.java:223)
	at com.intellij.concurrency.ApplierCompleter.d(ApplierCompleter.java:151)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1132)
	at com.intellij.concurrency.ApplierCompleter.b(ApplierCompleter.java:105)
	at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:534)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:479)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:94)
	at com.intellij.concurrency.ApplierCompleter.a(ApplierCompleter.java:116)
	at com.intellij.concurrency.ApplierCompleter.b(ApplierCompleter.java:96)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:143)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:234)
	at com.intellij.concurrency.ApplierCompleter.compute(ApplierCompleter.java:96)
	at java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:731)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
```

**Comment by kalessil** [comment]
@Hunternnm : I push a binary with fix into master branch, can you test it please?

How to:
- download https://github.com/kalessil/phpinspectionsea/blob/master/PhpInspectionsEA.jar
- in IDE go to plugins and above the plugins list click "Install from disk", point to the downloaded jar-file
- restart and share please feedback =)

**Comment by Hunternnm** [comment]
@kalessil Everything works perfectly! Thanks for the quick fix!

**Comment by kalessil** [comment]
Super, thank you for the fast feedback!

**Comment by Dragonrun1** [comment]
So the download fixes #584 but now it's flagging every method with a return type with false positive about using return type hint :-P 

![image](https://user-images.githubusercontent.com/1431541/31587154-473d289a-b191-11e7-9226-3135a6a28c63.png)

**Comment by kalessil** [comment]
At least some progress =), working on this case.
